ui/js/controllers/perf/alerts.js

```diff
@@ -563,7 +565,6 @@ perf.controller('AlertsCtrl', [
         };
 
         $scope.resetSummaryTitle = function () {
-            $scope.summaryTitle.promise.cancel();
```

Comment:
With this I completely removed the $q.defer()/async cancel usage from PhAlerts. Its usefulness was based on $http's timeout parameter (of type Promise), that could cancel an unresolved web request. I considered removing it outweighed the re implementation of this small Angular specific benefit.